packages/postgres/src/default-pool.ts

```diff
@@ -62,6 +62,9 @@ export const queryValidatedSingleColumnZeroOrOneRow =
 export const callValidatedRows = defaultPool.callValidatedRows.bind(defaultPool);
 export const callValidatedOneRow = defaultPool.callValidatedOneRow.bind(defaultPool);
 export const callValidatedZeroOrOneRow = defaultPool.callValidatedZeroOrOneRow.bind(defaultPool);
+export const queryRows = defaultPool.queryRows.bind(defaultPool);
+export const queryRow = defaultPool.queryRow.bind(defaultPool);
```

Comment:
As mentioned on #7739, we can't use `queryOneRow` like we originally planned on since that's already allocated to legacy code. I went with this for now, but `querySingleRow` is another option if we want to emphasize the "singleness" of the result.

Reply:
I'm really torn about this! I normally dislike having almost identical function names, but in this case I think that type hinting in the editor (and type checking in CI) will immediately make it clear if the wrong function is being used, so I'm leaning towards `queryRow`. I'd be happy with `querySingleRow` as well if you decide you prefer that.